### PR TITLE
Making most static strings in substrate.dev supporting multi-lingual

### PR DESCRIPTION
**Pull request by jimmychu0807**
Context: In docusaurus, strings put inside `<translate>...</translate>` tag will be uploaded to crowdin platform and be displayed in multi-lingual fashion. [See details here](https://docusaurus.io/docs/en/translation#enabling-translations-on-pages). This PR is to quote most static strings in substrate.dev to be `<translate/>` bounded.

Two pages not translated are: [Terms of Use](https://substrate.dev/terms) (low value), and [Videos](https://substrate.dev/videos) (this page is not linked from the main site, the videos are old, and the page should be re-make anyway).

Btw, I added [Substrate Frontier workshop](https://substrate.dev/frontier-workshop/#/) in tutorial section (i know this is a bad habit of doing multiple tasks in one PR 😅).

### CODE REVIEW

File: website/pages/en/index.js [hunk 1]
```diff
@@ -29,6 +29,9 @@ const translate = require('../../server/translate').translate
 class Index extends React.Component {
 	render() {
 		const { config: siteConfig, language = '' } = this.props
+	  const { baseUrl } = siteConfig;
```
Comment (kaichaosun) [inline website/pages/en/index.js:32]: Needs format

**Review by kaichaosun — APPROVED** [review]
👍 LGTM except a tiny comment

File: website/pages/en/index.js (hunk 1 above)
Comment (jimmychu0807) [reply website/pages/en/index.js]: Thanks. Have updated all JSs to use spacing for indentation

**Review by danforbes — COMMENTED** [review]
Thank you so much @jimmychu0807 and @kaichaosun 🙏🏻 Sorry I missed this PR but it looks like there's lot of good improvements in here 🎉

**Pull request merged**